apps/wing-console/console/app/tests/describe.ts

```diff
@@ -39,6 +39,10 @@ export const describe = (wingfile: string, callback: () => void) => {
     await page.getByTestId("loading-overlay").waitFor({ state: "hidden" });
 
     await page.waitForLoadState("domcontentloaded");
+
+    const blueScreen = page.getByTestId("blue-screen-of-death");
+
+    await expect(blueScreen).toBeHidden();
```

Comment:
I'd remove this check: if there's an error, no test will be able to pass anyways, so it's useless.

Reply:
done